pkg/kev/services.go

```diff
@@ -18,18 +18,15 @@ package kev
 
 import (
 	"encoding/json"
-	"errors"
 	"regexp"
-	"strings"
 
 	"github.com/appvia/kev/pkg/kev/config"
 	composego "github.com/compose-spec/compose-go/types"
-	"github.com/xeipuuv/gojsonschema"
 )
 
 func newServiceConfig(s composego.ServiceConfig) (ServiceConfig, error) {
 	config := ServiceConfig{Name: s.Name, Labels: s.Labels, Environment: s.Environment, Extensions: s.Extensions}
-	return config, config.validate()
+	return config, nil
```

Comment:
Do we still need to return an error?

Reply:
I would prefer keeping it just because updating the api requires more changes. I think this kind of cleanup should be done afterwards to minimize conflicts with master

Reply:
cool.